ansible/roles/candlepin/tasks/common/cp_debugging.yml

```diff
@@ -18,10 +18,13 @@
     - name: "Enable Tomcat debugging port in firewalld"
```

Comment:
Let's update the task name to something more general, like "Enable debugging ports in firewalld"

Reply:
Good catch, updated